### DISCUSSION TITLE
Amazon SP-API - extracting Global FBA Inventory Report, specifically the Upstream Storage Quantity

### DISCUSSION THREAD

**Discussion by jamesgbarton** [amzn/selling-partner-api-models#3789] [General]
Hi All,

Is there an API that aligns with Amazon's Warehouse and Distribution (AWD) inventory quantity (Upstream storage from FBA). Amazon appear to have only started providing this warehousing service and the information in the last 12 months. So it is relatively new.

Within Seller Central this data is within the fulfillment reports and called the **Gloabl FBA Inventory** report. The relevant data is labeled:
- Upstream storage available quantity
- Upstream storage reserved quantity

Currently having difficulty finding this data in other inventory reports or finding an existing API that links to this report/data. Any advice or support on this would be much appreciated.

@tspicer

**Comment by tspicer** [comment]
Hi @jamesgbarton ,

In looking at the Reporting API report types, there is no obvious match to the UI report. Maybe someone can provide some direction on which report would align best.

**Comment by erika89** [comment]
Hi ! I'm looking at this too. Did someone find it yet?
